src/routes/index.ts

```diff
@@ -53,11 +60,42 @@ export const createRoutes = (apis: Connections) => {
     })
   )
 
+  router.get(
+    '/balances/',
+    asyncErrorHandler(async (req, res) => {
+      const { account } = req.query
+
+      const networks = getConnectedNetworks(req)
```

Comment:
I guess better, put networs into the function instead req, because doesnt make sense do the hard dependency on req object